release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -764,6 +764,23 @@ For more information, see xref:../nodes/jobs/nodes-nodes-jobs.adoc#nodes-nodes-j
 
 {product-title} now supports the crun container runtime in Technology Preview. You can switch between the crun container runtime and the default container runtime as needed by using a `ContainerRuntimeConfig` custom resource (CR). For more information, see xref:../nodes/containers/nodes-containers-using.adoc#nodes-containers-runtimes[About the container engine and container runtime].
 
+[id="ocp-4-12-self-node-remediation-operator"]
+==== Self Node Remediation Operator enhancements
+
+{product-title} now supports control plane fencing on the Self Node Remediation Operator. In the event of node failure, you can follow remediation strategies on both worker nodes and control plane nodes. For more information, see xref:../nodes/nodes/eco-self-node-remediation-operator.adoc#control-plane-fencing-self-node-remediation-operator_self-node-remediation-operator-remediate-nodes[Control Plane Fencing].
```

Comment:
```suggestion
{product-title} now supports control plane fencing by the Self Node Remediation Operator. In the event of node failure, you can follow remediation strategies on both worker nodes and control plane nodes. For more information, see xref:../nodes/nodes/eco-self-node-remediation-operator.adoc#control-plane-fencing-self-node-remediation-operator_self-node-remediation-operator-remediate-nodes[Control Plane Fencing].
```

Reply:
> 

Thanks @anna-savina  I've applied the update.